src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -303,6 +306,72 @@ export default class RadarTargetModel {
         return [true, 'AMEND ALTITUDE'];
     }
 
+    /**
+     * Generate a string to be used for the first row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowOne
+     * @returns {string}
+     */
+    buildDataBlockRowOne() {
+        let dataBlockRowOne = `${this.aircraftModel.callsign}`;
+
+        if (this.aircraftModel.model.weightclass === 'H') {
+            dataBlockRowOne = `${this.aircraftModel.callsign}  ${this.aircraftModel.model.weightclass}`;
+        }
+
+        return dataBlockRowOne;
+    }
+
+    /**
+     * Generate a string to be used for the second row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowTwo
+     * @returns {string}
+     */
+    buildDataBlockRowTwo() {
+        const aircraftAltitude = round(this.aircraftModel.altitude * 0.01);
+        const aircraftSpeed = round(this.aircraftModel.groundSpeed * 0.1);
+
+        return `${leftPad(aircraftAltitude, 3)} ${leftPad(aircraftSpeed, 2)}`;
```

Comment:
An extra space is needed between these elements since the altitude length is 3. The timeshare has the first set of 4 characters, a space, then the second set of four characters. In both the primary and secondary sets, the second elements should not move left to right during the transition.
```
|---|---|
110  29
KSEA B738
|---|---|
```